src/presets/eu/src/eu-default.scss

```diff
@@ -169,6 +169,11 @@
 .ecl blockquote:not([class*='ecl-'], [class*='wt-']),
 .ecl blockquote:is([class*='ecl-u-']) {
   @extend %ecl-blockquote;
+}
+
```

Comment:
Why aren't we handling the margin top here in EU as we do in EC? I think it should be the same, it would otherwise result as an inconsistent behavior in the two themes.

Reply:
true, I just skipped it